pim/utils.py

```diff
@@ -113,17 +123,52 @@ def remove(self, name):
 
     def show(self):
         if len(self.required) == 0:
-            echo('\nNo packages installed')
+            echo('\nNo requirements.')
         else:
-            echo('\nInstalled packages: ')
-            with indent(4, quote='  -'):
-                for package in self.required:
-                    puts(package)
+            echo('\nCurrent requirements: ')
+            for package in self.required:
+                echo(' - ' + str(package))
```

Comment:
You may have missed this since I snuck it in on you right before you merged my PR, but I switched this to using `click.echo` instead of `clint.puts` because the click testing framework was not picking up the output from the `clint.puts`.